runtime/queries/lua/highlights.scm

```diff
@@ -219,3 +219,6 @@
 
 ;; Property
 (dot_index_expression field: (identifier) @variable.other.member)
+
+;; Method
+(function_call name: (dot_index_expression field: (identifier) @function.call))
```

Comment:
This should already be covered by the pattern on L192. The pattern above this takes precedence though. So Instead of adding this clause we should move the pattern above (L220-221) above the one that matches methods (L192)